common/src/unifyfs_client_rpcs.h

```diff
@@ -129,6 +130,30 @@ MERCURY_GEN_PROC(unifyfs_truncate_out_t,
                  ((int32_t)(ret)))
 DECLARE_MARGO_RPC_HANDLER(unifyfs_truncate_rpc)
 
+/* unifyfs_unlink_rpc (client => server)
+ *
+ * given an app_id, client_id, and global file id,
+ * unlink the file */
+MERCURY_GEN_PROC(unifyfs_unlink_in_t,
+                 ((int32_t)(app_id))
```

Comment:
Neither `app_id` nor `local_rank_idx` are necessary parameters. seems they are there to support the calls to `rm_cmd_unlink()` and `rm_cmd_truncate()`, but those functions don't use the parameters.

Reply:
Worth holding on to the app_id for future proofing for namespacing purposes of different mount points?

Reply:
good point. It's probably best to keep them. I think I even have updated code in the client API branch that needs that info.